client/src/app/api/api/api.service.ts

```diff
@@ -16,28 +16,39 @@
  * limitations under the License.
  */
 
-import {Inject, Injectable, Optional} from '@angular/core';
-import {HttpClient, HttpHeaders, HttpParams} from '@angular/common/http';
-import {Observable} from 'rxjs';
-import {Configuration} from '../configuration';
-import {BASE_PATH, INSTANCES, NUMBER_OF_INSTANCES, SYS_INFO} from '../variables';
-import {CustomHttpUrlEncodingCodec} from '../encoder';
-import {Instance} from '../model/instance';
-import {SysInfo} from '../model/sysInfo';
+import { Inject, Injectable, Optional } from '@angular/core';
+import { HttpClient, HttpHeaders, HttpParams } from '@angular/common/http';
+import { Observable } from 'rxjs';
+import { Configuration } from '../configuration';
+import {
+  BASE_PATH,
+  INSTANCES,
+  NUMBER_OF_INSTANCES,
+  SYS_INFO,
+  NEW_INSTANCE,
+  START_INSTANCE,
+  STOP_INSTANCE,
+  PAUSE_INSTANCE,
+  RESUME_INSTANCE,
+  DELETE_INSTANCE
+} from '../variables';
+import { CustomHttpUrlEncodingCodec } from '../encoder';
+import { Instance } from '../model/instance';
+import { SysInfo } from '../model/sysInfo';
 
 
 @Injectable({
   providedIn: 'root'
 })
 export class ApiService {
 
-  protected basePath = '';
+  public basePath = '';
```

Comment:
why did you change the visibility modifier to public? I only found new usage of the basePath in the test classes. Never alter production code just to make it easier to write test code. There are ways of accessing the variable in tests without making it public. Even if this doesn't work please create a getter function instead.

Reply:
getter method will be created

Reply:
Please resolve this issue